src/components/Admin/Messenger.jsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+
+import React, { useState } from "react";
+import Title from "./Title";
+
+const ADDRESSEES = ["volunteers", "mentors", "judges"];
+
+const Message = () => {
+  const [addressees, setAddressee] = useState([]);
+  const [subjectText, setSubjectText] = useState("");
+  const [messageBody, setMessageBody] = useState("");
+
+  const ToSelector = ({ selected, setSelected, allSelectorNames }) => {
+    const allButtonClasses = "ml-5 rounded-md mr-15 px-8 py-0.5 ";
+    const unselectedButtonClasses =
```

Comment:
don't make variables for styles, it gets messy

Reply:
alright I just added the new changes, let me know if there are any issues with them